### DISCUSSION TITLE
Can not run mobile app according to CONTRIBUTING.md guide

### DISCUSSION THREAD

**Discussion by qhhonx** [spacedriveapp/spacedrive#873] [Community Support]
I tried to run [spacedrive mobile-app](https://github.com/spacedriveapp/spacedrive/blob/main/CONTRIBUTING.md?rgh-link-date=2023-05-28T08%3A56%3A05Z#mobile-app) but got issue like [there](https://github.com/expo/expo/issues/17261) posted.

- For ios, I got `Unable to resolve "uuid/v5" from "../../node_modules/expo/build/environment/getInstallationIdAsync.js"`

- For android, I got `Error: crypto.getRandomValues() not supported. See https://github.com/uuidjs/uuid#getrandomvalues-not-supported, js engine: hermes`

Tried some actions referred by https://github.com/expo/expo/issues/17261, and "setting config.resolver.disableHierarchicalLookup to false in metro.config.js" works for me. 

So, do you guys have meet these similar issues ?

**Comment by utkubakir** [comment]
I can confirm this is producible. Created an issue for it #874, will be fixed soon.
